scripts/configs/env.mainnet.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
hi @ThanhNguyen140797 , plz have a look. Next time, Dev only needs to define the configurations here for both env mainnet and testnet. 

And require the utils/config to use them (https://github.com/CasperDash/casperdash-mobile-wallet/pull/40/files#diff-297c39b07d7667c9b8174682bf44d79d9376323ae2715eb250a51987baa33b14L1)